src/Nethermind/Nethermind.Serialization.Rlp/BlockDecoder.cs

```diff
@@ -62,15 +64,26 @@ public class BlockDecoder : IRlpValueDecoder<Block>, IRlpStreamDecoder<Block>
 
             rlpStream.Check(unclesCheck);
 
+            int withdrawalsSequenceLength = rlpStream.ReadSequenceLength();
```

Comment:
hmmm but for old blocks we won't have this sequence, so I'm wondering if it won't fail ReadSequenceLength();